svea_thesis/agentSetting[] agents
string communicationMethod
